source/Applications/Advanced/Downsample.m

```diff
@@ -11,13 +11,24 @@
 
 dataFile = [char(System.Environment.GetFolderPath(System.('Environment+SpecialFolder.CommonApplicationData'))),'/Zivid/Zivid3D.zdf'];
 disp(['Reading ZDF frame from file: ',dataFile]);
-[xyz,rgb,snr] = readZDF(dataFile);
+frame = Zivid.NET.Frame(dataFile);
+
+pointCloudZivid = frame.PointCloud();
```

Comment:
Why not just pointCloud?

Reply:
I see. Yes that is annoying. Did you check other Matlab samples? Maybe we have it already? If not, I guess this is ok.

Reply:
At one other sample it is called ptCloud. Not sure if that is better?

Reply:
Yes I think it is. Lets use that for Matlab.